.github/workflows/cclf-import-apply.yml

```diff
@@ -41,4 +41,13 @@ jobs:
           aws-region: ${{ vars.AWS_REGION }}
       - run: terraform init -reconfigure -backend-config=../../backends/$TF_VAR_app-$TF_VAR_env.s3.tfbackend
       - run: terraform apply -auto-approve
+      - name: notify slack # notify any terraform failiure via slack message in channel #ab2d-bcda-dpc-platform
+        id: slack
+        uses: slackapi/slack-github-action@v1.26.0
+        if: ${{ failure() }}
+        with:
+          channel-id: 'C04UG13JF9B' # slack channel to report terraform apply fail
+          slack-message: "Terraform apply failure\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}" # message that will be sent to the slack channel with the url
```

Comment:
I'd like the message to only include a link to the failing run of the workflow. See https://stackoverflow.com/a/70566764/589391 for constructing that link.

Please also drop these trailing comments.